### ISSUE TITLE
Themes borked

### ISSUE THREAD

**Issue by midget3111** [trigg/Rebuttal-Client#27]
Bork Bork. 

Only theme displayed is the default bubblegum.

**Comment by trigg** [comment]
Yeah, theme detection was written based on hosted-client. It hasn't been refactored to work after separation of client and server. Just need to populate the list on start.

**Comment by trigg** [comment]
Latest push fixes this in electron client

**Comment by midget3111** [comment]
Yep, this is fixed other than user icons. I'll open a new issue.
